test/kpod-mount.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+function teardown() {
+    cleanup_test
+}
+
+load helpers
+
+IMAGE="alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
```

Comment:
Might want to pick up `$STORAGE_OPTS` from `helpers.bash` here instead of specifying `--storage-driver` all the time.